### PR TITLE
LTC 5 cards

### PR DESCRIPTION
**Pull request by Simisays**

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/taunt_from_the_rampart.txt [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Name:Taunt from the Rampart
+ManaCost:3 R W
+Types:Sorcery
+A:SP$ Goad | Defined$ Valid Creature.YouDontCtrl | SubAbility$ DBEffect | RememberGoaded$ True | SpellDescription$ Goad all creatures your opponents control. Until your next turn, those creatures can't block. (Until your next turn, those creatures attack each combat if able and attack a player other than you if able.)
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/taunt_from_the_rampart.txt:4]: wrong Valid

File: forge-gui/res/cardsfolder/upcoming/taunt_from_the_rampart.txt [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+SVar:DBEffect:DB$ Effect | RememberObjects$ Valid Creature.YouCtrl | RememberObjects$ Remembered | StaticAbilities$ AllUnblockable | Duration$ UntilYourNextTurn | SubAbility$ DBCleanup
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/taunt_from_the_rampart.txt:5]: wrong remember

File: forge-gui/res/cardsfolder/upcoming/taunt_from_the_rampart.txt [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+SVar:AllUnblockable:Mode$ Continuous | EffectZone$ Command | AffectedZone$ Battlefield | Affected$ Card.IsRemembered | AddHiddenKeyword$ CARDNAME can't block. | Description$ Until your next turn, those creatures can't block.
+SVar:DBCleanup:DB$ Cleanup | ClearRemembered$ True
+AI:RemoveDeck:NonCommander
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/taunt_from_the_rampart.txt:8]: wrong tag

File: forge-gui/res/cardsfolder/upcoming/taunt_from_the_rampart.txt [hunk 4]
```diff
@@ -0,0 +1,8 @@
+Name:Taunt from the Rampart
+ManaCost:3 R W
+Types:Sorcery
+A:SP$ Goad | Defined$ Valid Creature.OppCtrl | SubAbility$ DBEffect | SpellDescription$ Goad all creatures your opponents control. Until your next turn, those creatures can't block. (Until your next turn, those creatures attack each combat if able and attack a player other than you if able.)
+SVar:DBEffect:DB$ Effect | RememberObjects$ Valid Creature.OppCtrl | StaticAbilities$ AllUnblockable | Duration$ UntilYourNextTurn | SubAbility$ DBCleanup
+SVar:AllUnblockable:Mode$ Continuous | EffectZone$ Command | AffectedZone$ Battlefield | Affected$ Card.IsRemembered | AddHiddenKeyword$ CARDNAME can't block. | Description$ Until your next turn, those creatures can't block.
+SVar:DBCleanup:DB$ Cleanup | ClearRemembered$ True
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/taunt_from_the_rampart.txt:7]: obsolete

**Comment by Simisays** [comment]
All right I'll close this one

**Pull request closed without merging**